packages/react/.storybook/Welcome/Welcome.js

```diff
@@ -15,22 +14,20 @@ import { Stack } from '../../src/components/Stack';
 
 export const Welcome = () => {
   return (
-    <div
-      className="welcome__container"
-      style={{
-        background: `url(${bg}) no-repeat center center fixed`,
-        backgroundSize: 'cover',
-      }}>
-      <h2 className="welcome__heading">@carbon/react</h2>
+    <div className="welcome__container">
+      <h2 className="welcome__heading">{PackageInfo.name}</h2>
       <h4 className="welcome__heading welcome__heading--subtitle">{`v${PackageInfo.version}`}</h4>
-      <Stack>
+      <Stack className="welcome__links" gap={5}>
         <Link
           href="https://carbondesignsystem.com/"
           className="welcome__link"
           renderIcon={ArrowRight}>
           Website
         </Link>
         <Link
+          // TODO: Should the `href` be `PackageInfo.repository.url` instead?
+          // The current value links to the package's directory, not the
+          // repository. The link's text says "GitHub repo".
```

Comment:
I think it should link to the package directory as this storybook is specific to the react package